### PR TITLE
Update include module to support sort/filter

### PR DESCRIPTION
**Pull request by hammaddev0**

### CODE REVIEW

File: lib/fat_ecto/query/group_by.ex [hunk 1]
```diff
@@ -118,30 +159,42 @@ defmodule FatEcto.FatQuery.FatGroupBy do
...
+  defp _group_by(queryable, group_by_param, opts) do
     field = FatHelper.string_to_existing_atom(group_by_param)
 
-    from(
-      q in queryable,
-      group_by: field(q, ^field),
-      select_merge: %{
-        "$group" => %{
-          ^group_by_param => field(q, ^field)
+    if opts[:binding] do
```
Comment (tanweerdev) [inline lib/fat_ecto/query/group_by.ex]: should match on value of binding ie `opts[:binding] == :last` or `opts[:binding] == :first`

File: test/query/aggregate_test.exs [hunk 2]
```diff
@@ -185,30 +185,26 @@ defmodule Query.AggregateTest do
         "fat_hospital" => %{
           "$join" => "$full",
           "$order" => %{"id" => "$desc"},
-          "$where" => %{"name" => "Saint"}
+          "$where" => %{"name" => "Saint"},
+          "$binding" => :first
```
Comment (tanweerdev) [inline test/query/aggregate_test.exs]: Frontend is also supposed to send `$binding`?

File: lib/fat_ecto/query/group_by.ex (hunk 1 above)
Comment (hammaddev0) [reply lib/fat_ecto/query/group_by.ex]: ok

File: test/query/aggregate_test.exs (hunk 2 above)
Comment (hammaddev0) [reply test/query/aggregate_test.exs]: yes

Comment (tanweerdev) [reply test/query/aggregate_test.exs]: Frontend cant send binging in `atom` form. it can only send strings

**Pull request merged**